### ISSUE TITLE
Regular Release Cycles

### ISSUE THREAD

**Issue by mattiasaxell** [okfde/froide#166]
So @robinharms is getting to work on the system Swedish FROIDE-instance in about a week. Something that would be really helpful for us in maintaining the system would be a continuous release cycle.

When can the code be frozen for a stable release @stefanw ? And what do you imagine to be the ideal continous schedule for release cycles? To start scheduling regular releases would be great to avoid incompatibility issues when doing e.g. security updates to existing systems etc? @ryankanno @fin 

I have seen info somewhere about 3.6.1 and something in ReadTheDocs documentation about 4.0.0-alpha documentation. More info and structure would be much appreciated for the expansion of FROIDE! :)

**Comment by ryankanno** [comment]
:+1:

**Comment by mattiasaxell** [comment]
How's it going for you @ryankanno ? I will ask the ones who helped set it up for us to see if they can contribute with documentation, they did not use the ubuntu + vagrant-setup.

**Comment by rugk** [comment]
Oh dear, last release was v5 [in 2018](https://github.com/okfde/froide/releases/tag/v5.0.0) – I doubt this is "intended"?
So you should [tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) your [releases](https://help.github.com/articles/about-releases/), so they can be seen [on GitHub](https://help.github.com/articles/creating-releases/).

**Comment by stefanw** [comment]
Froide is Open Source, but I don't work *for* froide, I work *on* froide. I work *for* FragDenStaat.de which is not a software project, but an FOI campaign organisation. Our organisational goals (e.g. political impact) have a higher priority then release management of the underlying software project. 

We don't use releases internally, the master branch of all associated projects is deployed. We don't have the development capacity to have a proper release cycle. Froide also doesn't have many downstream users that rely on regular releases (might be a self-fulfilling prophecy though).
I can imagine more joint stewardship with other downstream users and then better release management, but right now there are no other major contributors to the project that make this feasible.

In summary: there will not be releases just to scratch someones software engineering itch.
